src/EnvelopedData.js

```diff
@@ -932,6 +932,8 @@ export default class EnvelopedData
 				 */
 				_this.recipientInfos[index].value.recipientEncryptedKeys.encryptedKeys[0].encryptedKey = new asn1js.OctetString({ valueHex: result });
 				//endregion
+
+				return {ecdhPrivateKey};
```

Comment:
I'm not returning the private key directly in case you need to return something else in the future. I don't have a strong preference.